run-dainik.js

```diff
@@ -0,0 +1,29 @@
+const cheerio = require('cheerio')
```

Comment:
@shrijan-siris no module export here? these files not being used at the moment? If you are using these files for local test only, you can test writing tests for `manual-scrapper.js`. 
Let's prioritize `test driven development`